better-custom-response/custom-responses.ts

```diff
@@ -117,6 +117,34 @@ const customResponses: CustomResponse[] = [
         icon_emoji: ':kadokawa:',
         username: 'KADOKAWA',
     },
+	{
+		input: [/デニム/],
+		outputFunction: (input: string[]) => {
+			let lane = [1,2,3,4,5,6,7];
+			for(let i = 6; i > 0; i--){
+				let r = Math.floor(Math.random() * (i + 1));
+				let tmp = lane[i];
+				lane[i] = lane[r];
+				lane[r] = tmp;
+			}
```

Comment:
```suggestion
			const lane = shuffle([1, 2, 3, 4, 5, 6, 7]);
```
みたいにするとええんちゃう？(世の中には便利関数があるので)
ところでこれだけではだめで、4行目あたりに
```
import { shuffle } from 'lodash';
```
みたいなのを入れないと動きません(は？)